README.md

```diff
@@ -32,7 +32,7 @@ To uninstall TinyTeX, use the command:
 choco uninstall tinytex
 ```
 
-## Scoop packages
+## Scoop package
```

Comment:
Just a nitpick
```suggestion
## Scoop Package
```

Reply:
I did the same as for chocolatey package
https://github.com/yihui/tinytex-releases/blob/master/README.md#L19

No uppercase P 😄

Reply:
Do for both. 😉

Reply:
It is just a convention right ? 
As we used Sentence case for the first header, I prefer not to use Title Case just for these two other header. I prefer to leave that out for now.